pkg/network/setup/podnic.go

```diff
@@ -210,8 +210,10 @@ func (l *podNIC) discoverAndStoreCache() error {
 func (l *podNIC) PlugPhase2(domain *api.Domain) error {
 	precond.MustNotBeNil(domain)
 
-	if err := l.domainGenerator.Generate(); err != nil {
-		log.Log.Reason(err).Critical("failed to create libvirt configuration")
+	if l.domainGenerator != nil {
```

Comment:
This is extremely hacky.
I think you had this handled [here](https://github.com/kubevirt/kubevirt/blob/a79137bee1cf345fb64a7c77cdc2667dec5c6bb7/pkg/network/setup/network.go#L111) in a previous version of this change. Is there a reason not to handle it there?

Reply:
Its possible, I was under the impression this is the prefered way.

I changed it to have Slirp ifaces filtered at PodNIC creation instead.